### DISCUSSION TITLE
How many Super Streams are we allowed to create?

### DISCUSSION THREAD

**Discussion by mikegsoares** [rabbitmq/rabbitmq-stream-dotnet-client#317] [Q&A]
Hi, all!
I would like to know if there's a number Super Streams we are allowed to create by stream and if creating more [partitions] it impacts performance by means.
Thanks in advance

**Comment by acogoluegnes** [comment]
There is no theoretical limit to either the number of super streams or the number of partitions (streams) in a super stream. But having a few dozen or hundred individual streams or a super stream with 10 partitions is already plenty to work with.

These are just general numbers, it really depends on your use cases and workloads: ingress rates, number of consumers, number of connections, etc. We can be more specific if you tell us more about those.

**Comment by jonnepmyra** [comment]
> Hi, all! I would like to know if there's a number Super Streams we are allowed to create by stream and if creating more [partitions] it impacts performance by means. Thanks in advance

We're using SuperStreams with 20 partitions in production without any problems. Adding more partitions to our SuperStreams is how we increase concurrency. In need of more throughput, we add more partitions.
